templates/definition/vehicle/ioBroker.bmw.yaml

```
@@ -0,0 +1,64 @@
+template: ioBroker.bmw
+products:
+  - brand: ioBroker.bmw
```

Besser Description als Marke. Sonst taucht das auf der Webseite als Markennamen auf.

```suggestion
  - description:
      generic: ioBroker.bmw
```